### ISSUE TITLE
Extend Annex A. (normative) Attribute Definitions

### ISSUE THREAD

**Issue by petrvanekrobe** [mvrdevelopment/spec#210]
Extend the [Annex A. (normative) Attribute Definitions](https://github.com/mvrdevelopment/spec/blob/main/gdtf-spec.md#annex-a-normative-attribute-definitions)  with information from [gdtf_attributes_with_description](https://github.com/mvrdevelopment/spec/blob/main/gdtf_attributes_with_description.json).

**Comment by reneberhorst** [comment]
I agree! We should create a third column in the Annex A that will be called "explanation" and will reflect the text of the jason.explanation...
